$NetBSD: patch-aa,v 1.6 2008/05/09 22:53:21 dsainty Exp $

Disable HAVE_ABSTRACT_SOCKETS as a workaround for bugs in the abstract socket
implementation, see:
  http://bugs.freedesktop.org/show_bug.cgi?id=15884

--- Xtranssock.c.orig	2008-05-09 02:23:07.000000000 +1200
+++ Xtranssock.c	2008-05-10 09:41:28.000000000 +1200
@@ -159,7 +159,7 @@
 #endif
 
 #ifdef linux
-#define HAVE_ABSTRACT_SOCKETS
+/*#define HAVE_ABSTRACT_SOCKETS*/
 #endif
 
 #define MIN_BACKLOG 128
@@ -294,11 +294,6 @@
  * that don't have IPv6 support.
  */
 #if defined(IPv6) && defined(AF_INET6)
-static const struct in6_addr local_in6addr_any = IN6ADDR_ANY_INIT;
-#pragma weak in6addr_any = local_in6addr_any
-#ifndef __USLC__
-#pragma weak getaddrinfo
-#endif
 static int haveIPv6 = 1;
 #endif
 
